src/Repository/SkillRepository.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use Doctrine\ORM\EntityRepository;
+
+class SkillRepository extends EntityRepository
+{
+    /**
+     * Finds all availebles skills for autocomplete that connected user has not yet chosen.
+     */
+    public function getSkillsForAutocomplete(string $term, Adherent $user)
+    {
+        $qbUserSkills = $this
+            ->createQueryBuilder('us')
+            ->select('us.name')
+            ->leftJoin('us.summary', 'cv')
+            ->andWhere('cv.member = :user')
+        ;
+
+        $qb = $this->createQueryBuilder('s');
+        $qb
+            ->distinct('s.name')
+            ->leftJoin('s.summary', 'summary')
+            ->where('s.name LIKE :name')
+            ->andWhere('summary.member <> :member')
+            ->andWhere($qb->expr()->notIn('s.name', $qbUserSkills->getDQL()))
+            ->setParameters([
+                'name' => '%'.$term.'%',
```

Comment:
should be `$term.'%'` only, we want to autocomplete skills that start with the given term.